tests/test_feedexport.py

```diff
@@ -1587,16 +1590,60 @@ def test_export_items(self):
         header = self.MyItem.fields.keys()
         yield self.assertExported(items, header, rows, settings=Settings(settings))
 
+    @defer.inlineCallbacks
+    def test_export_file_size_trigger(self):
+        items = [
+            self.MyItem({'foo': 'bar1', 'egg': 'spam1', 'baz': 'quux1'}),
+            self.MyItem({'foo': 'bar2', 'egg': 'spam2', 'baz': 'quux2'}),
+        ]
+        fmt_to_limits = {
+            'json': ('40B', '90B'),
+            'jsonlines': ('40B', '90B'),
+            'xml': ('100B', '160B'),
+            'csv': ('30B', '40B'),
+            'marshal': ('30B', '70B'),
+            'pickle': ('50B', '110B'),
```

Comment:
Looks good. However, I still think it would be best to be more precise, to use a pair of values that only differs in 1 byte.

In fact, I think you can be more precise without actually having to guess the right value to put here. Which is great, because I suspect for pickle and marshal that value may change depending on the version of Python that you use.

When you run the `1B` test, you can use the `exported_data` to tell what exact size the first file has, so using that size as maximum should still cause 2 files (`self.file_size >= self.max_file_size`), while that size plus 1 should cause a single file. So, instead of hardcoding sizes here, you could use the output of `self.exported_data(items, settings)` for `1B` for the tests based on higher sizes.

Reply:
Got it!